src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java

```diff
@@ -922,4 +936,31 @@ private String validateListAPI(String orgId, SunbirdApiRequest request, CbPlanSe
 
         return errMsg;
     }
+
+    private String getDesignationForUser(String profileDetails) throws IOException {
```

Comment:
don't throw the exception... just log and return empty string...

Reply:
Made the implementation